tests/config/sp_catch_brace.cfg

```diff
@@ -0,0 +1 @@
+sp_catch_brace = force
```

Comment:
BTW, I know we have a *lot* of files that are missing newlines, but can we please try to stop adding new ones?

Reply:
Thank you!